app/controllers/games_controller.rb

```diff
@@ -13,7 +13,12 @@ def create
 
   def show
     @game = Game.find_by_id(params[:id])
-    render text: 'Not Found', status: :not_found if @game.blank?
+    if @game.blank?
+      render text: 'Not Found', status: :not_found
+      return
+    end
+    flash[:notice] = 'Black King Check!' if @game.check?('black')
+    flash[:notice] = 'White King Check!' if @game.check?('white')
```

Comment:
@koglesby, never mind. Was able to reproduce it. Strange. Looks like it only triggers sometime. Hmm. Should I just overrule Rubocop on this one?